ledger-core/v2/virtual/object/object.go

```diff
@@ -55,6 +55,8 @@ type Info struct {
 	PotentialMutablePendingCount   uint8
 
 	objectState State
+
+	pendingTable pendingTable
```

Comment:
I wonder if we could avoid double mapping, we already have KnownRequests map, maybe we could have this maps in one place?

Reply:
missed that one, fixed